analyses/DESeq_analysis/run-DESeq-analysis.R

```diff
@@ -0,0 +1,195 @@
+# Author: Sangeeta Shukla and Alvin Farrel
+# Date: June 2021
+# Function: 
+# 1. summarize Differential expression from RNASeq data
+# 2. tabulate corresponding P-value
+
+# Example run: DESeq
+# Rscript analyses/DESeq/run-DESeq-analysis.R 1 1
+# This was compares 1 cancer group (Combined or Cohort specific) with one GTEx tissue type. The input arguments are the indices of the comparison of the cancer groups and gtex sub tissues to be compared. In v6, there are 107 cancer groups vs 54 GTEx tissues
+
+# Read arguments from terminal
+args <- commandArgs(trailingOnly = TRUE)
+
+# Load required libraries
+suppressPackageStartupMessages({
+  library(ggplot2)
+  #library(DESeq2)
+})
+
+
+HIST_index <- args[1]   #assign first argument to Histology index
+GTEX_index <- args[2]   #assign second argument to GTEx index
+
+    
+
+
+
+#Load histology file
+hist <- read.delim("histologies.tsv", header=TRUE, sep = '\t')
```

Comment:
R version > 4 has `stringsAsFactors = FALSE` by default, but the Docker image has R 3.6.0 with `stringsAsFactors = TRUE` by default. Read strings as factors will cause line 119 not be able to subset expected rows.

```suggestion
hist <- read.delim("histologies.tsv", header=TRUE, sep = '\t', stringsAsFactors = FALSE)
```